crates/orderbook/src/api/post_order.rs

```diff
@@ -263,6 +263,19 @@ pub fn create_order_response(
     }
 }
 
+#[utoipa::path(
+    post,
+    path = "/v1/create_order",
+    request_body = OrderCreation,
+    responses(
+        (status = 200, description = "Order has been accepted.", body = OrderUid),
+        (status = 400, description = "Error during order validation."),
```

Comment:
Is it possible to define this on an error struct returned by this endpoint instead?
That way it could probably be enforced that every error must have a description.

Reply:
Maybe we can implement https://docs.rs/utoipa/latest/utoipa/trait.IntoResponses.html on `AddOrderError`